desktop/src/main/java/bisq/desktop/components/paymentmethods/JapanBankTransferForm.java

```diff
@@ -0,0 +1,395 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.desktop.components.paymentmethods;
+
+import bisq.common.util.Tuple2;
+import bisq.common.util.Tuple3;
+import bisq.common.util.Tuple4;
+
+import bisq.desktop.components.InputTextField;
+import bisq.desktop.components.AutocompleteComboBox;
+import bisq.desktop.util.FormBuilder;
+import bisq.desktop.util.Layout;
+import bisq.desktop.util.validation.LengthValidator;
+import bisq.desktop.util.validation.RegexValidator;
+import bisq.desktop.util.validation.JapanBankTransferValidator;
+import bisq.desktop.util.validation.JapanBankBranchCodeValidator;
+import bisq.desktop.util.validation.JapanBankBranchNameValidator;
+import bisq.desktop.util.validation.JapanBankAccountNumberValidator;
+import bisq.desktop.util.validation.JapanBankAccountNameValidator;
+import bisq.desktop.components.paymentmethods.data.JapanBankData;
+
+import bisq.core.account.witness.AccountAgeWitnessService;
+import bisq.core.locale.Res;
+import bisq.core.locale.TradeCurrency;
+import bisq.core.payment.PaymentAccount;
+import bisq.core.payment.JapanBankAccount;
+import bisq.core.payment.payload.PaymentAccountPayload;
+import bisq.core.payment.payload.JapanBankAccountPayload;
+import bisq.core.util.BSFormatter;
+import bisq.core.util.validation.InputValidator;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javafx.scene.control.TextField;
+import javafx.scene.layout.GridPane;
+import javafx.scene.control.Label;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.control.RadioButton;
+import javafx.util.StringConverter;
+
+import static bisq.desktop.util.FormBuilder.*;
+import static bisq.desktop.util.GUIUtil.getComboBoxButtonCell;
+
+public class JapanBankTransferForm extends PaymentMethodForm
+{
+    private final JapanBankAccount japanBankAccount;
+    protected ComboBox<String> bankComboBox, bankAccountTypeComboBox;
+    private InputTextField bankAccountNumberInputTextField;
```

Comment:
noob here. don't be mad at me if I'm wrong :)
shouldn't it be?
`private` InputTextField bankAccountNumber, InputTextField;`

Reply:
Nope :-)